.gitignore

```diff
@@ -0,0 +1,3 @@
+app.config
```

Comment:
You should add the .vs folder to .gitignore (or use a template) and un-commit all the stuff in .vs

Reply:
Also it looks like you added a lot of stuff in obj/ before putting in in .gitignore you you'll need to uncommit those as well - let me know if you need a hand with that!